.github/workflows/gh-pages.yaml

```diff
@@ -0,0 +1,36 @@
+name: gh-pages
+
+on:
+  push:
+    branches: master
+    paths: docs/**
+
+jobs:
+  gh-pages:
+    runs-on: ubuntu-latest
+    container:
+      image: debian:testing
+
+    defaults:
+      run:
+        shell: sh
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          apt-get -y update
+          apt-get -y install --no-install-recommends python3-pip
+          pip3 install --upgrade --requirement tools/requirements.txt
+
+      - name: Generate docs
+        run: |
+          cd docs
+          poxy --verbose
```

Comment:
[`poxy`](https://github.com/marzer/poxy) requires a relatively recent version of Doxygen (the docker image I was using had v1.9.2). This image might need it installed or updated for poxy to work correctly.

Reply:
Is the [`doxygen`](https://packages.debian.org/testing/doxygen) package enough or does it need extra components like [`graphviz`](https://packages.debian.org/testing/graphviz) (containing the `dot` executable)?

Reply:
Doxygen alone should suffice :)